### ISSUE TITLE
Set referer/useragent before downloading

### ISSUE THREAD

**Issue by jsalatiel** [DjazzLab/ocdownloader#24]
Please add a way to set the referer url and/or useragent before start some download.

**Comment by DjazzLab** [comment]
Yep it makes sense. Do you want to set it manually via the admin UI ? Or automatically ?

**Comment by jsalatiel** [comment]
I was thinking about extra options:
Some text box for useragent and some text box for referer under the text boxes for 
Basic Auth User : and Basic Auth Password 

Of course it would be nice to pre-set the user agent field in some preferences.

**Comment by DjazzLab** [comment]
Ok thanks for the additional information!
I can add this feature in a future release (maybe next week).

I'll let you know ;-)
